packages/experiments/README.md

```diff
@@ -1,51 +1,85 @@
 # Experiments
 
-Private `__experimental` APIs that are not [exposed publicly plugin authors](https://make.wordpress.org/core/2022/08/10/proposal-stop-merging-experimental-apis-from-gutenberg-to-wordpress-core/#respond).
+`@wordpress/experiments`  enables sharing private `__experimental` APIs across `@wordpress` packages without
+[publicly exposing them to WordPress extenders](https://make.wordpress.org/core/2022/08/10/proposal-stop-merging-experimental-apis-from-gutenberg-to-wordpress-core/#respond).
 
-This package acts as a "dealer" that only allows WordPress packages to use the experimental APIs.
+## Getting started
 
-Each package needs to start by registering itself:
+Every `@wordpress` package wanting to privately access or expose experimental APIs must opt-in to `@wordpress/experiments`:
 
 ```js
-const { register } =
+// In packages/block-editor/experiments.js:
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/experiments';
+export const { lock, unlock } =
 	__dangerousOptInToUnstableAPIsOnlyForCoreModules(
-		'<CONSENT STRING>',  // See index.js, this may change without notice.
-		'@wordpress/blocks'
+		'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.',
+		'@wordpress/block-editor' // The package opting in
 	);
 ```
 
-The function name communicates that plugins are not supposed to use it. To make double and triple sure, the first argument must be that exact consent string, and the second argument must be a known `@wordpress` package that hasn't opted in yet – otherwise an error is thrown.
+Each package may only opt-in once. The function name communicates that plugins are not supposed to use it. To make double and triple sure, the first argument must be that exact consent string, and the second argument must be a known `@wordpress` package that hasn't opted in yet – otherwise an error is thrown.
```

Comment:
```suggestion
Each package may only opt in once. The function name communicates that plugins are not supposed to use it.

The function will throw an error if the following conditions are not met:

1. The first argument must exactly match the required consent string: `'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.'`. 
2. The second argument must be a known `@wordpress` package that hasn't yet opted into `@wordpress/experiments`
```

Optional copy update. Just to make it clear that the first arg must be an exact match of the `requiredConsent` value.

Also optional, because no one cares 😄 "opt in" and "opt out" are the verbs. "opt-in" and "opt-out" are adjectives. 

🍺

Reply:
Oh that's great, thank you!